modules/network/src/test/java/org/apache/ignite/internal/network/serialization/MarshallableTest.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.network.serialization;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.UnpooledByteBufAllocator;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.embedded.EmbeddedChannel;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Map;
+import org.apache.ignite.internal.network.direct.DirectMessageWriter;
+import org.apache.ignite.internal.network.netty.ConnectionManager;
+import org.apache.ignite.internal.network.netty.InboundDecoder;
+import org.apache.ignite.network.NetworkMessage;
+import org.apache.ignite.network.TestMessageSerializationRegistryImpl;
+import org.apache.ignite.network.TestMessagesFactory;
+import org.apache.ignite.network.serialization.MessageSerializationRegistry;
+import org.apache.ignite.network.serialization.MessageSerializer;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+/**
+ * Tests marshallable serialization.
+ */
+public class MarshallableTest {
+    /** {@link ByteBuf} allocator. */
+    private final UnpooledByteBufAllocator allocator = UnpooledByteBufAllocator.DEFAULT;
+
+    /** Registry. */
+    private final MessageSerializationRegistry registry = new TestMessageSerializationRegistryImpl();
+
+    /**
+     * Tests that marshallable object can be serialized along with it's descriptor.
+     */
+    @Test
+    public void testMarshallable() {
+        TestMessagesFactory msgFactory = new TestMessagesFactory();
+
+        ChannelHandlerContext ctx = Mockito.mock(ChannelHandlerContext.class);
```

Comment:
Same as for `toList()` and `assertXXX()`: it seems reasonable to statically import all `Mockito` methods because everyone understands that, as we are in the context of tests, `mock()` means `Mockito.mock()` and `when()` comes from `Mockito`. At the same time, this reduces the noise and clutter in the code. Also, `doReturn(a).when(b).call()` reads as English prose, while `Mockito.doReturn(a).when(b).call()` certainly does not.